resources/views/users/verify.blade.php

```diff
@@ -16,8 +16,10 @@
     along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
 --}}
 @extends('master', [
+    // Verification doesn't inherit from App\Controller, thus these variables aren't set. Thus we set them here:
     'currentSection' => 'error',
     'currentAction' => '401',
+    'currentSectionHue' => section_to_hue_map('error'),
```

Comment:
wouldn't it be easier to just call `section_to_hue_map` inside the layout itself :question: